docs/core/logger.md

```diff
@@ -604,7 +604,28 @@ By default all registered loggers will be modified. You can change this behavior
 ---8<-- "examples/logger/src/cloning_logger_config.py"
 ```
 
-**What's the difference between `append_keys` and `extra`?**
+### How can I add standard library logging attributes to a log record?
+
+The Python standard library log records contains a [large set of atttributes](https://docs.python.org/3/library/logging.html#logrecord-attributes){target="_blank"}, however only a few are included in Powertools Logger log record by default.
+
+You can include any of these logging attributes as key value arguments (`kwargs`) when instantiating `Logger` or `LambdaPowertoolsFormatter`.
+
+You can also add them later anywhere in your code with `append_keys`, or remove them with `remove_keys` methods.
+
+=== "collect.py"
+
+    ```python hl_lines="3 8 10"
+    ---8<-- "examples/logger/src/append_and_remove_keys.py"
+    ```
+=== "Example CloudWatch Logs excerpt"
+
+    ```json hl_lines="6 15-16"
+    ---8<-- "examples/logger/src/append_and_remove_keys.json"
+    ```
+
+For log records originating from Powertools `Logger`, the `name` attribute will be the same as `service`, for log records coming from standard library logger, it will be the name of the logger (i.e. what was used as name argument to `logging.getLogger`).
```

Comment:
```suggestion
For log records originating from Powertools Logger, the `name` attribute will be the same as `service`, for log records coming from standard library logger, it will be the name of the logger (i.e. what was used as name argument to `logging.getLogger`).
```